packages/mars-theme/src/components/post.js

```diff
@@ -146,8 +146,7 @@ const Content = styled.div`
 
   figure {
     margin: 24px auto;
-    /* next line overrides an inline style of the figure element. */
-    width: 100% !important;
```

Comment:
I hope nothing breaks removing that `!important`... :sweat_smile:

Reply:
I could not see notice anything breaking after removing it and as @SantosGuillamot has pointed out almost nobody uses unmodified mars-theme so I don't think this would break anything.

To clarify, this is being removed as AMP prohibits the use of `!important` in CSS.